src/trigger.py

```diff
@@ -36,6 +36,8 @@ def _run_trigger(args, build_config, db):
     if args.skip_pull:
         print("Not updating repo")
     else:
+        node_list = get_node_by_commit_id(
+            kernelci.build.get_branch_head(build_config))
```

Comment:
The `node_list` variable is not used in this commit, so I think it makes more sense to squash it with the next one which checks if it's empty.